tests/e2e/testfiles/storage_csm_powerscale_observability_top_custom_cert.yaml

```diff
@@ -1,488 +1,488 @@
-apiVersion: storage.dell.com/v1

```

Comment:
What happened to this file, why is it being shown as every line removed and re-added? I can't see a difference between the two versions.